neqo-bin/src/client/mod.rs

```diff
@@ -554,6 +557,18 @@ pub async fn client(mut args: Args) -> Res<()> {
             remote_addr,
         );
 
+        let migration = if args.shared.qns_test == Some("connectionmigration".to_owned()) {
```

Comment:
Why is this not possible?

```suggestion
        let migration = if args.shared.qns_test == Some("connectionmigration") {
```

I know that `qns_test` is a `String`, but you should be able to `PartialEq` it with `&'static str` perfectly well.

Reply:
Maybe then...

```suggestion
        let migration = if args.shared.qns_test.as_deref() == Some("connectionmigration") {
```